test/test_app.sh

```diff
@@ -31,12 +31,21 @@ group :development, :test do
   gem "test-unit-rails", path: "/source/"
 end
 GEMFILE
+if [[ "${RAILS_VERSION}" =~ ^4 ]]; then
+  cat >> Gemfile <<GEMFILE
+gem 'loofah', '< 2.21.0'
+GEMFILE
+fi
+
 # For Rails 5 Gemfile
 sed -i'' -e "s/gem 'chromedriver-helper'/gem 'webdrivers'/" Gemfile
-# Rails 5.0 doesn't work with sqlite >= 1.4
-if [ "${RAILS_VERSION}" = "5.0" ]; then
+# Rails 4.x and 5.0 doesn't work with sqlite >= 1.4
+if [ $(echo "${RAILS_VERSION} < 5.1" | bc) -eq 1 ]; then
   sed -i'' -e "s/gem 'sqlite3'/gem 'sqlite3', '~> 1.3.6'/" Gemfile
 fi
+if [[ "${RAILS_VERSION}" =~ ^4 ]]; then
```

Comment:
Could you add a comment what you described in the PR description here too?
It'll be easy to refer on maintenance.

Reply:
Did so.